### ISSUE TITLE
Describe Each UI Element

### ISSUE THREAD

**Issue by CrazyMarvin** [Crazy-Marvin/Morse#21]
Every UI element should be [described](https://developer.android.com/guide/topics/ui/accessibility/apps) for TalkBack users
